tests/devops/Core/DevOps/StaticAnalyse/PHPStan/Rules/data/PropertyNativeTypeRule/PropertiesCorrectlyTyped.php

```diff
@@ -26,14 +19,12 @@ class PropertiesCorrectlyTyped
     public $callableProperty;
 
     /**
-     * @param string $promotedStringPropertyWithDeprecation
      * @param resource $promotedResourceProperty
      * @param callable $promotedCallableProperty
      */
     public function __construct(
         public string $promotedStringProperty,
-        /** @deprecated tag:v6.7.0 - Will be natively typed */
-        public $promotedStringPropertyWithDeprecation,
+        public string $promotedStringPropertyWithDeprecation,
```

Comment:
you can remove this, as this case is not needed anymore. 

but maybe, you can revert the changes at all for now. as it would like to this here: https://github.com/shopware/shopware/pull/6839

Reply:
okay, i will close this PR 👍